### ISSUE TITLE
Guided Tours - keyboard navigation breaks out of tour

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#40007]
### Steps to reproduce the issue
See video recording - keyboard navigation only - all key presses highlighted on screen by external software
Creating seperate issue for each bug report so that they dont get lost inside a bigger report (again)

As shown in the video in some steps you can navigate to unexpected buttons

https://user-images.githubusercontent.com/1296369/222925152-8a76d318-3707-4465-a83d-a72d5c102c6d.mp4

The requirement to use a mouse to successfully use the Guided Tours component is an accessibility failure and breaks our commitment to providing an accessible way to use Joomla

**Comment by obuisard** [comment]
I don't know why, none of the videos are visible, making it hard to understand the issue here.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40007">issues.joomla.org/tracker/joomla-cms/40007</a>.</sub>

**Comment by brianteeman** [comment]
No idea why they dont show on the issue tracker. They do show perfectly on github

**Comment by obuisard** [comment]
> No idea why they dont show on the issue tracker. They do show perfectly on github

My bad, I should have used GitHub instead...

**Comment by brianteeman** [comment]
addressed elsewhere
